### ISSUE TITLE
Feedback - Some suggestions

### ISSUE THREAD

**Issue by gperdomor** [passivestar/godot-minimal-theme#8]
Hi @passivestar...

Thank you for this awesome theme, it's great, but I think can be improved a little bit because in some parts of the UI it's too flat, or in this case, too minimal, I don't known if it's intended 😂

Here are 2 screenshots with the default theme and other 2 with the minimal theme for comparison... Basically in the FileSystem tree, the arrow paths are missing, also some "containers" have no borders and can be a little confusing because everything looks like a label and not a clickable resource (Like the Sprite 2D in the create new node modal, below Recents label)

I hope you can consider this feedback 😄 

BTW, the screenshots are for Godot 4.3 beta 2 and minimal theme high ppi v1.5

<img width="1792" alt="Screenshot 2024-07-03 at 8 00 44 PM" src="https://github.com/passivestar/godot-minimal-theme/assets/371939/d4cc29e9-197f-43da-9971-4a82d0f3040b">
<img width="1792" alt="Screenshot 2024-07-03 at 8 00 46 PM" src="https://github.com/passivestar/godot-minimal-theme/assets/371939/48505078-0f4b-4006-9b70-0a082dfa631d">
<img width="1792" alt="Screenshot 2024-07-03 at 8 03 26 PM" src="https://github.com/passivestar/godot-minimal-theme/assets/371939/ef753263-08be-4675-901f-e64bfe38f446">
<img width="1792" alt="Screenshot 2024-07-03 at 8 03 31 PM" src="https://github.com/passivestar/godot-minimal-theme/assets/371939/3ecb8f2e-8a97-4b2d-9294-79c644f9974b">

**Comment by passivestar** [comment]
> I don't known if it's intended 😂

It's not, it's just what we have to deal with considering godot's current theming limitations

<img width="592" alt="image" src="https://github.com/passivestar/godot-minimal-theme/assets/60579014/65bd0937-d98d-493b-a905-76551ca0bd8d"><br>

Godot theming still lacks a lot, I'd very much like to have clear visible separation for those containers but since they affect all docks I had to make a choice. Fixing this will require a big amount of work on the engine side, probably something to explore in 4.4

> the arrow paths are missing

some people mentioned that, I might bring those back but I'd like to see what it'll look like with just vertical lines, like it's done in vscode. might be a good compromise on minimalism/function

Thanks for the feedback! <3

**Comment by gperdomor** [comment]
:o... I understand... thank you for the quick response 👍🏻

**Comment by Horops** [comment]
Yeah I came here to say this also. The lack of line (arrow?) paths in the Scene and FileSystem windows makes it *really* hard to visualize what goes where.  
I attached a screenshot for example (though not the best example since the tree is really small in this case so not that hard to follow but in a bigger project it gets unwieldy fast)
I understand it's currently a Godot limitation though as PassiveStar said.
Hope the org can make this change possible sometime soon.
![Untitled](https://github.com/user-attachments/assets/6541d253-7c8f-4ab1-be5c-0260f7456d7d)

**Comment by passivestar** [comment]
There's now a relationship line for the currently selected list item since https://github.com/passivestar/godot-minimal-theme/commit/57a0d73988e8675e7a4421c4f62f00e9fbc42573 in the latest theme version. This is a half measure to make lists clean while providing some guidance. If I were to draw lines for all items some lists would become too noisy, that's especially noticeable on lists that don't need any guides to begin with, such as the sidebars of editor/project settings

Panels for sidebars were added in https://github.com/passivestar/godot-minimal-theme/commit/67ff2af7f03270543d7e78ec066b42ee5c24f545 and the PR to make them possible on the engine side was merged today. Wait for the next dev release of Godot for them to appear (4.4-dev6)
